docs/index-server.md

```diff
@@ -21,7 +21,7 @@
 
 Carbon currently prunes and caches all block/blocklet datamap index information into the driver for
 normal table, for Bloom/Index datamaps the JDBC driver will launch a job to prune and cache the
-datamaps in executors.
+indexes in executors.
```

Comment:
Remove datamap keyword in Line No.22, Line No.23, Line no.105, Line no.123, Line No.155  and Line no.159

Reply:
that is done in PR3738

Reply:
ok

Reply:
handled in 3738